docs/pages/docs/core-concepts/filtering.mdx

```diff
@@ -8,55 +8,67 @@ import HeartIcon from "@heroicons/react/solid/HeartIcon";
 
-## Filter Syntax
+Turborepo supports a `--filter` flag that lets you **select the workspaces you'd like to execute your task in**.
+
+![With a filter on shared, only the shared package runs test](../../../public/images/docs/with-filter.png)
 
-Filters specify combinations of workspaces, directories, and git commits to act as entrypoints
-for execution.
+You can use it to:
 
-Multiple filters can be combined to select distinct sets of targets. Additionally, filters
-can also exclude targets. A target that matches any filter and is not explicitly excluded will
-be in the final entrypoint selection.
+- Filter by [workspace name](#filter-by-workspace-name)
+- Filter by [workspace directory](#filter-by-directory)
+- Include [dependents](#include-dependents-of-matched-workspaces) and [dependencies](#include-dependencies-of-matched-workspaces) of matched workspaces
+- Execute tasks from the [workspace root](#the-workspace-root)
+- Filter by [changes in git history](#filter-by-changed-workspaces)
+- [Exclude workspaces](#excluding-workspaces) from selection
 
-Turborepo's filter syntax is based on pnpm's filter syntax, so it should feel familiar to pnpm users.
+Turborepo will run each task against each matched workspace, ensuring that any dependent tasks are run first, according to the `pipeline` specification in [`turbo.json`](/docs/reference/configuration#pipeline).
 
-### Filter by workspace
+## Filter Syntax
 
-Supports exact matches (`--filter=my-pkg`), and globs (`--filter=*my-pkg*`) are supported.
-Scoped workspaces (`@foo/bar`) can be abbreviated to just the workspace name (`bar`) as long as the workspace name
-is unique within the monorepo (e.g. `@foo/bar` exists, but `@baz/bar` does not).
+### Multiple filters
 
-The monorepo's root can be selected using the token `//`.
+You can specify multiple filters by adding multiple `--filter` to the command:
+
+```sh
+turbo run build --filter=my-pkg --filter=my-app
+```
+
+### Filter by workspace name
+
+Supports exact matches (`--filter=my-pkg`), and globs (`--filter=*my-pkg*`).
 
 ```sh
 # Build 'my-pkg', letting `turbo` infer task dependencies
 # from the pipeline defined in turbo.json
 turbo run build --filter=my-pkg
 
-# Build '@foo/bar', letting `turbo` infer task dependencies
+# Build '@acme/bar', letting `turbo` infer task dependencies
 # from the pipeline defined in turbo.json
-turbo run build --filter=@foo/bar
+turbo run build --filter=@acme/bar
 
 # Build all workspaces that start with 'admin-', letting turbo infer task
 # dependencies from the pipeline defined in turbo.json
 turbo run build --filter=admin-*
+```
 
-# Run the format script from the root "package.json" file:
-turbo run format --filter=//
+#### Scopes
+
+Some monorepos prepend their workspace names with a scope, such as `@acme/ui` and `@acme/app`. As long as the scope (`@acme`) is unique across the codebase, you may omit it from filters.
```

Comment:
What happens when it isn't unique, just of my knowledge?

Reply:
Unsure! Might throw an error, or not be found?